config.go

```diff
@@ -53,6 +54,17 @@ func (conf *Config) Validate() error {
 	return nil
 }
 
+//SaveConfigFile writes config to supplied filename
+func (conf *Config) SaveConfigFile(filename string) error {
```

Comment:
Give that it's already a method on `Config`, this could just be called `SaveFile` or `WriteFile`.